pkg/serializer/split/split.go

```diff
@@ -163,3 +169,14 @@ func marshal(m marshaler.Marshaler, mType MarshalType) ([]byte, error) {
 		return m.MarshalJSON()
 	}
 }
+
+func handleDroppedPayloads() {
+	log.Warnf("Payloads could not be splitted and were dropped")
+	splitterPayloadDrops.Add(1)
+
```

Comment:
nit: blank line
You could also inline this at this point if you prefer

Reply:
Yes, we doesn't need a function anymore